### ISSUE TITLE
Floating panel stops working when.. 

### ISSUE THREAD

**Issue by d0x360** [clipto-pro/Android#86]
The floating panel will stop working if you use the accessibility button to enable and disable the service.  Clicking the green bar that says show panel won't work and neither will my quick panel shortcut.   BTW it would be nice if we could open the panel from the notification (all variants)

I thought it would be a convenient way to hide and show the window but it's not lol.

You have to disable the service, force stop clipto, open clipto, enable the service and then the overlay will work again.

I don't know if it can be made to work using the accessibility button but if it can't a warning should be added explaining that you shouldn't use the button.

![Screenshot_20210927-115615_Clipto](https://user-images.githubusercontent.com/14056857/134943941-92860e24-23ed-4f88-a7e1-f93cbb09c8f8.jpg)
![Screenshot_20210927-115842_Accessibility](https://user-images.githubusercontent.com/14056857/134943991-19f7f535-96aa-45f2-862e-b1254ec8bb6e.jpg)
